dev/requirements.txt

```diff
@@ -50,7 +50,7 @@ black==22.6.0
 py
 
 # Spark Connect (required)
-grpcio==1.48.1
```

Comment:
I commited this by accident - trying to revert it now. But OSS should probably change this requirement at some point because it didn't work for me with this version.